src/error.rs

```diff
@@ -1,11 +1,12 @@
 #[derive(Fail, Debug, PartialEq)]
+
 pub enum PacksError {
     #[fail(display = "last_admin_of_group")]
     LastAdmin,
     #[fail(display = "error_deleting_members")]
     ErrorDeletingMembers,
-    #[fail(display = "profile_not_found")]
-    ProfileNotFound,
+    #[fail(display = "profile_not_found: {}. ({})", _0, _1)]
+    ProfileNotFound(String, String),
```

Comment:
The meaning of these two strings is rather unclear, so you could consider making them named fields:
```
    ProfileNotFound { user_uuid: String, msg: String },
```
This would make constructing the variant a bit more verbose, though, so you'd probably want to introduce some helper function to construct it. I'm not sure it's worth it, since wherever this variant is used it's not hard to tell what the fields are.